shared-doc/helm-deploy-project.adoc

```diff
@@ -61,28 +106,43 @@ deploy:
 ----
 endif::requires-http-route[]
 
-This will create a new deployment on OpenShift and deploy the application.
+This will create a new deployment on {cloud-platform} and deploy the application.
 
 If you want to see all the configuration elements to customize your deployment you can use the following command:
 [source,options="nowrap",subs="+attributes"]
 ----
 $ helm show readme {helmChartName}
 ----
 
-
+ifdef::openshift[]
 Get the URL of the route to the deployment.
 
 [source,options="nowrap",subs="+attributes"]
 ----
 $ oc get route {helm-app-name} -o jsonpath="{.spec.host}"
 ----
 Access the application in your web browser using the displayed URL.
+endif::[]
+ifdef::kubernetes[]
+To be able to connect to our application running in Kubernetes from outside, we need to set up a port-forward to the `{helm-app-name}` service created for us by the Helm chart.
+
+This service will run on port `8080`, and we set up the port forward to also run on port `8080`:
 
+[source,options="nowrap",subs="+attributes"]
+----
+kubectl port-forward service/{helm-app-name} 8080:8080
+----
+The server can now be accessed via `http://localhost:8080` from outside Kubernetes. Note that the command to create the port-forward will not return, so it is easiest to run this in a separate terminal.
+
+endif::[]
 [NOTE]
 ====
 The Maven profile named `openshift` is used by the Helm chart to provision the server with the quickstart deployed on the root web context, and thus the application should be accessed with the URL without the `/{artifactId}` path segment after `HOST:PORT`.
 ====
 
-ifdef::post-helm-install-actions[]
-include::{post-helm-install-actions}[leveloffset=+1]
-endif::post-helm-install-actions[]
+ifdef::openshift,post-helm-install-actions-openshift[]
```

Comment:
I see it now. Both files were getting included, the '+' helps. 

The extra 'splt' is needed here since helm-deploy-project.adoc is being used by both OpenShift and Kubernetes. For the prerequisites, this is not required since there are build-and-run-the-quickstart-with-kubernetes.adoc and build-and-run-the-quickstart-with-openshift.adoc files which each include the prerequisite file for their environment